src/highdicom/sr/templates.py

```diff
@@ -3519,6 +3632,297 @@ def __init__(
                 item.RelationshipType = relationship_type.value
                 self.append(item)
 
+    def _generate_projection_radiography_descriptors(
+        self,
+        dataset: Dataset
+    ) -> Sequence[ContentItem]:
+
+        """
+        :dcm:`TID 1603 <part16/chapter_A.html#sect_TID_1603>`
+        Image Library Entry Descriptors for Projection Radiography
+
+        Parameters
+        ----------
+        pydicom.Dataset
+            Metadata of a projection radiology image
+
+        Returns
+        -------
+        Sequence[highdicom.sr.ContentItem]
+            SR Content Items describing the image
+
+        """  # noqa: E501
+        patient_orientation = dataset.PatientOrientation
+        pixel_spacing = dataset.ImagerPixelSpacing
+        descriptors = [
+            TextContentItem(
+                name=CodedConcept(
+                    value='111044',
+                    meaning='Patient Orientation Row',
+                    scheme_designator='DCM'
+                ),
+                value=patient_orientation[0],
+                relationship_type=RelationshipTypeValues.HAS_ACQ_CONTEXT,
+            ),
+            TextContentItem(
+                name=CodedConcept(
+                    value='111043',
+                    meaning='Patient Orientation Column',
+                    scheme_designator='DCM'
+                ),
+                value=patient_orientation[1],
+                relationship_type=RelationshipTypeValues.HAS_ACQ_CONTEXT,
+            ),
+            NumContentItem(
+                name=CodedConcept(
+                    value='111026',
+                    meaning='Horizontal Pixel Spacing',
+                    scheme_designator='DCM'
+                ),
+                value=pixel_spacing[1],
+                relationship_type=RelationshipTypeValues.HAS_ACQ_CONTEXT,
+                unit=CodedConcept(
+                    value='mm',
+                    meaning='millimeter',
+                    scheme_designator='UCUM'
+                )
+            ),
+            NumContentItem(
+                name=CodedConcept(
+                    value='111066',
+                    meaning='Vertical Pixel Spacing',
+                    scheme_designator='DCM'
+                ),
+                value=pixel_spacing[0],
+                relationship_type=RelationshipTypeValues.HAS_ACQ_CONTEXT,
+                unit=CodedConcept(
+                    value='mm',
+                    meaning='millimeter',
+                    scheme_designator='UCUM'
+                )
+            )]
+
+        # Not yet implemented from TID 1603 - unclear how to derive data:
+        # - ImageView, ImageViewModifiers
+        # - PositionerPrimaryAngle, PositionerSecondaryAngle
+
```

Comment:
Since these are all optional (user defined), it's fine being opinionated and leaving them out.

```suggestion
```